afids_utils/tests/test_ext.py

```diff
@@ -211,17 +208,21 @@ def test_save_fcsv_valid_template(
             # Check contents
             for idx, row in enumerate(output_fcsv):
                 assert (row["x"], row["y"], row["z"]) == (
-                    str(afids_coords[idx].x),
-                    str(afids_coords[idx].y),
-                    str(afids_coords[idx].z),
+                    str(afids_set.afids[idx].x),
+                    str(afids_set.afids[idx].y),
+                    str(afids_set.afids[idx].z),
                 )
 
-    @given(afids_coords=afid_coords(bad_range=True))
-    def test_invalid_num_afids(self, afids_coords: list[AfidPosition]) -> None:
+            # Check to see if file can be loaded with afids-utils
+            test_load = AfidSet.load(out_fcsv_file.name)
+            assert isinstance(test_load, AfidSet)
```

Comment:
I _think_ we could even `assert test_load == afids_set`

Reply:
Ahh yeah, this test can be simplified quite a bit with the new `afid_sets` strategy